alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: alby-nostr-wallet-connect
+name: Nostr Wallet Connect
+tagline: The power of the zaps at the tip of your fingers.
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
+version: "0.1.0"
+port: 58000
+description: >-
+  Connect your LND node over Nostr.
+
+  Zap directly from within supported apps like Damus & Amethyst.
+
+  Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
+
+  Damus (currently only on Testflight) video guide: https://streamable.com/bareih
```

Comment:
Here's how the formatting will look in the app store:

<img width="867" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/4adda679-ddc0-4c43-8c63-338d7ba6aab2">

If you want to, you can use 2 blank lines to create additional whitespace, and can use `-` to represent bullet points. Meaning something like this is possible:

<img width="867" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/250ee6a6-4d2a-4336-b5f4-48df617adf42">